src/widgets/_widgets.scss

```diff
@@ -29,38 +29,64 @@ dialog {
 }
 
 window:not(.popup) {
+    box-shadow: 0 0 0 1px $toplevel-border-color;
+
     > decoration {
-        @include border-shadow;
         @include shadow(4);
 
         &:backdrop {
-            @include border-shadow;
             @include shadow(2)
         }
     }
 
     &.tiled {
         > decoration {
-            @include border-shadow;
             @include shadow(1);
         }
     }
+
+    headerbar.titlebar {
```

Comment:
Is there a reason this is specific to headerbar.titlebar and no just .titlebar?

Reply:
I'm also not sure that it makes sense to nest this here, since `.popup` isn't going to have a headerbar widget… I think it would make more sense for this to be top-level.

Reply:
`.titlebar` only is fine, but these styles are specifically for titlebars on Windows and don't apply to dialogs